src/main/java/seedu/address/logic/commands/AddStudentToTutorialCommand.java

```diff
@@ -68,7 +68,7 @@ public CommandResult execute(Model model) throws CommandException {
             model.setStudent(studentToEdit, editedStudent);
         }
 
-        return new CommandResult(MESSAGE_SUCCESS, NavigationMode.TUTORIAL);
+        return new CommandResult(MESSAGE_SUCCESS, NavigationMode.STUDENT);
```

Comment:
```suggestion
        return new CommandResult(MESSAGE_SUCCESS, NavigationMode.UNCHANGED);
```

I think it would be better if we used `NavigationMode.UNCHANGED` instead